python/lsst/ts/cbp/component.py

```diff
@@ -362,11 +412,11 @@ async def check_cbp_status(self):
         None
 
         """
-        self.encoder_status.AZIMUTH = bool(await self.send_command("AAstat=?"))
-        self.encoder_status.ELEVATION = bool(await self.send_command("ABstat=?"))
-        self.encoder_status.MASK_SELECT = bool(await self.send_command("ACstat=?"))
-        self.encoder_status.MASK_ROTATE = bool(await self.send_command("ADstat=?"))
-        self.encoder_status.FOCUS = bool(await self.send_command("AEstat=?"))
+        self.encoder_status.AZIMUTH = bool(int(await self.send_command("AAstat=?")))
```

Comment:
Surely this should be the motion fields? Else why cast to boolean?

Reply:
Its the status of the encoder.

Reply:
Got it. So once the doc string for the class is updated and the values initialized to False instead of 0, this should be be clear.